### PR TITLE
Added confirmation to domain server webpage before restarting

### PR DESCRIPTION
**Pull request by seefo**
Should fix https://highfidelity.fogbugz.com/f/cases/5520/Move-restart-button-in-domain-server-to-right-side-of-nabber-and-require-confirmation

Test Plan:
1. Start domain server
2. Visit domain server webpage
3. On the navbar you should see a button on the right that says to Restart the domain server
4. Click the restart button and verify that a warning dialog pops up that asks you if you're sure that you'd like to restart your domain server

### CODE REVIEW

**Comment by CainFoool** [comment]
Testing

File: domain-server/resources/web/js/domain-server.js [hunk 1]
```diff
@@ -33,9 +33,17 @@ $(document).ready(function(){
       return this.href == url;
   }).parent().addClass('active');  
 
-  $('body').on('click', '#restart-server', function(e){
-    $.get("/restart");
-    showRestartModal();
+  $('body').on('click', '#restart-server', function(e) {    
+    swal( {
+      title: "Are you sure?",
+      text: "This will restart your domain server, causing your domain server to be briefly offline.",
```
Comment (birarda) [inline domain-server/resources/web/js/domain-server.js]: "causing your domain to be briefly offline"

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1016/HighFidelity-Beta-PR10787-d834a05fd9ef2e8ce7d4acd4216d43d1f9fe662d.exe

**Comment by CainFoool** [comment]
Test | Result
-------- | --------
Start domain server | Pass
Visit domain server webpage | Pass
On the navbar you should see a button on the right that says to Restart the domain server | Pass
Click the restart button and verify that a warning dialog pops up that asks you if you're sure that you'd like to restart your domain server | Pass

Let me know if there's anything else you'd like me to test.
###### Intel i5 7600K @ 4Ghz, 16GB DDR4 2133Mhz Corsair Vengeance, Asus Strix GTX 1050Ti 3GB, HTC Vive

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1024/HighFidelity-Beta-PR10787-d87a55d6e628071b336bac892db8242895d10065.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1024/HighFidelity-Beta-PR10787-d87a55d6e628071b336bac892db8242895d10065.dmg

**Pull request merged**